docs/src/docs/asciidoc/user/clustered-cache.adoc

```diff
@@ -0,0 +1,61 @@
+---
+---
+= Clustered Cache
+ifndef::sourcedir[]
+include::common.adoc[]
+endif::sourcedir[]
+
+ifdef::notBuildingForSite[]
+include::menu.adoc[]
+endif::notBuildingForSite[]
+
+== Introduction
+
+To create clustered caches, you will need a cache manager with clustering capabilities so that the caches can consume the clustered storage.
+
+=== Creating cache manager with clustering capabilities
+
+For creating the cache manager with clustering support, you will need to provide the clustering service configuration:
+
+[source,java,indent=0]
+----
+include::{sourcedir}/clustered/client/src/test/java/org/ehcache/clustered/client/docs/GettingStarted.java[tag=clusteredCacheManagerExample]
+----
+
+<1> returns the `org.ehcache.config.builders.CacheManagerBuilder` instance;
+<2> Use the `ClusteringServiceConfigurationBuilder`{empty}'s static method `.cluster(URI)` for connecting the cache manager to the clustering storage at the
+    URI specified that returns the clustering service configuration builder instance.
+    Sample URI provided in the example is pointing to the clustered storage with clustered storage identifier *my-application*; the query-param `auto-create`
+    creates the clustered storage in the server if it doesn't already exist.
+<3> Returns a fully initialized cache manager that can be used to create clustered caches.
+<4> Close the cache manager.
+
+=== Cache manager using server side resources
+
+Server side storage resources can be used when creating cache manager with clustering support:
```

Comment:
Here the example is not only about using, it is also about configuring the available storage to the clustered bits.